governance/votes/Executive vote - January 24, 2022.md

```diff
@@ -0,0 +1,63 @@
+
+The express purpose of this proposal is to prevent FLAP auctions temporarily until the auction/burn mechanism can be improved.
+
+
+If this executive proposal passes, the following **changes** will occur within the Maker Protocol:
+* The existing [System Surplus Buffer](https://manual.makerdao.com/parameter-index/core/param-system-surplus-buffer) lerp function will be disabled.
+* The [System Surplus Buffer](https://manual.makerdao.com/parameter-index/core/param-system-surplus-buffer) parameter will be set to 250 million DAI.
+
+
+**Voting for this executive proposal will place your MKR in support of the changes and additions outlined above.**
+
+Unless otherwise noted, the changes and additions listed above are subject to the [GSM Pause Delay](https://community-development.makerdao.com/en/learn/governance/param-gsm-pause-delay). This means that if this executive proposal passes, the changes and additions listed above will only become active in the Maker Protocol after the GSM Pause Delay has expired. The GSM Pause Delay is currently set to **48 hours**.
+
+If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Maker Protocol.
+
+---
+
+## Proposal Details
+
+### Temporarily prevent FLAP auctions and MKR Burn
+
+As per this [urgent signal request](https://forum.makerdao.com/t/urgent-signal-request-stop-mkr-burning/12806), the following changes will be made if this executive proposal passes.
+* The existing [System Surplus Buffer](https://manual.makerdao.com/parameter-index/core/param-system-surplus-buffer) lerp function will be disabled.
+* The [System Surplus Buffer](https://manual.makerdao.com/parameter-index/core/param-system-surplus-buffer) parameter will be set to 250 million DAI.
+
+This change is being proposed to prevent FLAP auctions temporarily until the auction/burn mechanism can be improved. Further details on this issue and the origin of this proposal can be found in the links below:
+* [Flappy Friday - CLIP and FLAP Analysis](https://forum.makerdao.com/t/flappy-friday-clip-and-flap-analysis/12790)
+* [So is someone going to say anything about yesterday’s FLAP auctions?](https://forum.makerdao.com/t/so-is-someone-going-to-say-anything-about-yesterdays-flap-auctions/12781)
+* [Stop MKR Burning](https://forum.makerdao.com/t/urgent-signal-request-stop-mkr-burning/12806)
+
+## Review
+
+Community debate on these topics can be found on the MakerDAO [Governance forum](https://forum.makerdao.com/). Please review any linked threads to inform your position before voting.
+
+Additionally, these changes may have been discussed further in recent Governance calls. [Video](https://www.youtube.com/playlist?list=PLLzkWCj8ywWNq5-90-Id6VPSsrk4OWVan) for these calls is available to review.
```

Comment:
This bit of the template doesn't really apply because it's urgent. Could be removed if desired.

Reply:
Good call, removed.

Reply:
We should create an urgent template sometime.